src/web-server.js

```diff
@@ -236,6 +238,60 @@ app.delete("/services/:serviceName", function (req, res) {
     });
 });
 
+app.patch("/scopes", function (req, res) {
```

Comment:
This is not what was discussed in the Github issue. The plan is to support import map scopes with `PATCH /import-map.json`. We do not need a new api to support it.